src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -620,6 +620,10 @@
     <value>Enable Markdown (.md) preview</value>
     <comment>Do not loc "Markdown".  Do you want this feature on / off</comment>
   </data>
+  <data name="FileExplorerPreview_ToggleSwitch_Preview_Monaco.Header" xml:space="preserve">
+    <value>Enable source code preview (Monaco)</value>
```

Comment:
@crutkas
Should we replace "monaco" here with file extension examples too like in our docs?

Reply:
no end user facing language should lead with reference Monaco.  
how about: `Enable developer files preview (Example: .cpp, .py, ...)`

Reply:
Done